rfcs/RFC-Uploads.md

```diff
@@ -0,0 +1,401 @@
+# User Uploads
+
+## For the cellxgene Data Portal
+
+**Authors:** [Trent Smith](mailto:trent.smith@chanzuckerberg.com)
+
+**Approvers:** [Arathi Mani](mailto:arathi.mani@chanzuckerberg.com), [Timmy Huang](mailto:thuang@chanzuckerberg.com),
+[Marcus Kinsella](mailto:mkinsella@chanzuckerberg.com), [Eduardo Lopez](mailto:elopez@chanzuckerberg.com), [Brian Raymore](mailto:braymor@chanzuckerberg.com)
+[Sara Gerber](mailto:sara.gerber@chanzuckerberg.com)
+
+## tl;dr
+
+A user can upload files of any size to Data Portal using either a shared link from their cloud provider or directly from their browser.
+
+## Glossary
+
+- CSP(Cloud Storage Provider) - Cloud storage services such as Dropbox or Google Drive.
+- Shared Link - A link from a CSP that gives the bearer of that link access to private data.
+- DP(Data Portal) - The cellxgene Data Portal.
+- Small File - a file < 5 GB in size.
+- Large File - a file > 5 GB in size.
+
+## Problem Statement | Background
+
+One of the primary goals of the DP is to allow users to upload their own files. These files consist of data sets as well
+as attestation files. These data sets can vary in size from a few MB to several GB. Users may want to upload from their local machine or their CSP. For users that want to upload from their local machine, they must be able to resume an
```

Comment:
Can you add attestation files to the glossary.  It's probably common knowledge, but I'm not sure what it mean.

Reply:
We need to review this with Shawn - but my expectation is that there will be one copy of the attestation document on the portal and only a boolean (wasAccepted) or a version string (accepted v 1.1) associated with the collection.